src/DataTable/TableControlBar.jsx

```diff
@@ -34,20 +36,14 @@ const TableControlBar = ({
       <div className="pgn__data-table-status">
         <div className="pgn__data-table-status-left">
           <SmartStatus
-            isSelectable={isSelectable}
-            isFilterable={isFilterable}
-            numberOfSelectedRows={selectedFlatRows.length}
-            toggleAllRowsSelected={toggleAllRowsSelected}
-            filterNames={filterNames}
-            pageSize={pageSize}
             itemCount={itemCount}
-            resetAllFilters={resetAllFilters}
+            {...rest}
```

Comment:
What's getting passed to all of these components? (`BulkActions` etc)

Reply:
bulkActions, mostly, to maintain the ability to define that on DataTable. I think my idea was to allow people to pass whatever props they wanted down to these components. I don't know how useful that actually is, though.